packages/truffle-db/package.json

```diff
@@ -56,10 +55,12 @@
   "dependencies": {
     "@gnd/graphql-tools": "^4.0.5-fix.0",
     "@gnd/jsonschema2graphql": "^1.0.15",
-    "@truffle/artifactor": "^4.0.31",
+    "@truffle/artifactor": "^4.0.32",
+    "@types/pouchdb-adapter-leveldb": "^6.1.3",
```

Comment:
FYI `@types/` packages can go in `devDependencies`, since they're only used during development and don't get distributed with the package